Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -49,7 +50,7 @@ static ByteString bookmarks_file_path()
     return builder.to_byte_string();
 }
 
-BrowserWindow::BrowserWindow(WebView::CookieJar& cookie_jar, Vector<URL> const& initial_urls)
+BrowserWindow::BrowserWindow(WebView::CookieJar& cookie_jar, Vector<URL> const& initial_urls, StringView const man_file)
```

Comment:
This is fine, but in the future we don't do `const` on value-parameters. It's purely documentation at the call site and limits the implementation.